src/app.js

```diff
@@ -32,6 +32,33 @@ setup();
 /*                                CLI Initializtion                          */
 /*****************************************************************************/
 
+//Initilize start up logo
+console.log(
```

Comment:
Put this in a method and call it here. Put that method in a different file to keep app.js file clean